lib/cocoapods/validator.rb

```diff
@@ -55,6 +59,19 @@ def file
       @linter.file
     end
 
+    # Returns a list of platforms to lint for a given Specification
+    #
+    # @param [Specification] spec
+    #         The specification to lint
+    #
+    # @return [Array<Platform>] platforms to lint for the given specification
+    #
+    def platforms_to_lint(spec)
```

Comment:
minor nit (ignore if you want to): `spec` param is unnecessary here, `validator` provides an attribute

Reply:
I think `spec` here can also be a subspec, and not just the root spec